x-pack/plugins/apm/public/components/app/settings/agent_explorer/index.tsx

```diff
@@ -99,27 +101,36 @@ export function AgentExplorer() {
       <EuiSpacer size="s" />
       <EuiFlexItem grow={false}>
         <EuiTitle>
-          <EuiFlexGroup gutterSize="s">
-            <EuiFlexItem grow={false}>
-              <h2>
-                {i18n.translate('xpack.apm.settings.agentExplorer.title', {
-                  defaultMessage: 'Agent explorer',
-                })}
-              </h2>
-            </EuiFlexItem>
-            <EuiFlexItem grow={false}>
-              <TechnicalPreviewBadge icon="beaker" />
-            </EuiFlexItem>
-          </EuiFlexGroup>
+          <EuiFlexItem>
```

Comment:
do you need the extra `EuiFlexItem`

EuiFlexItem -> EuiTitle -> **EuiFlexItem** -> EuiFlexGroup?

Reply:
Also, <EuiTitle> shouldn't be inside the EuiFlexGroup ? 🤔

Reply:
You are right, it's enough to have
`EuiFlexItem -> EuiTitle ->  EuiFlexGroup`